script.js

```diff
@@ -0,0 +1,164 @@
+
+class RenderManager {
+    list = new ListManager();
+    submitButton = document.getElementById("addMission");
+    clearAllButton = document.getElementById("clear-all");
+    unFinishedConteiner = document.getElementById("unfinishedMissions")
+    finishedConteiner = document.getElementById("finishedMissions")
```

Comment:
there is a problem with saving this as a field of the class,
can you think on the problem and suggest a solution?

Reply:
yes
all of this should not be part of the class because like that they will be dependent (or bind) to the instance of RenderManager

i move them out to const out of the class.

another option is to do those as "static".

Reply:
discuused